org.enjoyingfoss.FOSStriangulator.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.enjoyingfoss.FOSStriangulator
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions :
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: FOSStriangulator
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: FOSStriangulator
+    buildsystem: simple
+    build-commands:
+      - install -D FOSStriangulator /app/bin/FOSStriangulator
+      - install -D FOSStriangulator.jar /app/share/FOSStriangulator.jar
+      - install -Dm644 Instructions.png /app/share/Instructions.png
+      - install -Dm644 processing.png /app/share/processing.png
+      - install -Dm644 icon.png /app/share/icon.png
+      - install -Dm644 org.enjoyingfoss.FOSStriangulator.desktop /app/share/applications/org.enjoyingfoss.FOSStriangulator.desktop
+      - install -Dm644 org.enjoyingfoss.FOSStriangulator.metainfo.xml /app/share/metainfo/org.enjoyingfoss.FOSStriangulator.metainfo.xml
+      - install -Dm644 org.enjoyingfoss.FOSStriangulator.svg  /app/share/icons/hicolor/scalable/apps/org.enjoyingfoss.FOSStriangulator.svg
+      - install -Dm644 org.enjoyingfoss.FOSStriangulator-symbolic.svg /app/share/icons/hicolor/symbolic/apps/org.enjoyingfoss.FOSStriangulator-symbolic.svg
+    sources:
+      - type: archive
+        sha256: 6c95f19006a770e957754b5afc9f8624a04142eb87e5cdf78017cb6b5729309e
+        url: https://github.com/FOSStriangulator/FOSStriangulator/releases/download/v1.0.0/FOSStriangulator.tar.xz
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.desktop
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.metainfo.xml
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator.svg
+      - type: file
+        path: org.enjoyingfoss.FOSStriangulator-symbolic.svg
```

Comment:
Ideally these files should be part of the `tar.xz` above and maintained in https://github.com/FOSStriangulator/FOSStriangulator

Reply:
@barthalion Should be fixed with the latest commit